docs/Halogen/Menu/Query.md

```diff
@@ -0,0 +1,13 @@
+## Module Halogen.Menu.Query
+
+#### `MenuQuery`
+
+``` purescript
+data MenuQuery a next
```

Comment:
Yes. It would be nice to have those have a peekable value too. I've raised this as an enhancement here https://github.com/slamdata/purescript-halogen-menu/issues/4